src/routes/company.svelte

```diff
@@ -54,7 +53,7 @@
     <div class="md:w-1/3">
       <h3 class="text-2xl md:text-5xl">We're here <br /><b>to help</b></h3>
       <p class="mt-4 text-darkgray text-base">
-  We are building a new set of products and services that have the potential to transform the software community. These products and services provide development teams the guidance and capabilities to, not only, highly perform when building software, but to future proof their products, and create constant and consistent value over time.
+  We are building a new set of products and services that will transform the software development. Don't partner with a framework or service that decreases your productivity over time and creates a mountain of technical debt. At hyper, we are building tools that require a different mindset and can give you a hyper focus on your problem domain.
```

Comment:
As your code base expands, hyper actually _increases_ productivity while _decreasing_ your technical debt.  At hyper, we build tools that require a different mindset.  Tools that provide a hyper focus on your problem domain.